apps/remote_control/lib/lexical/remote_control/search/store/backends/khepri.ex

```diff
@@ -0,0 +1,150 @@
+defmodule Lexical.RemoteControl.Search.Store.Backends.Khepri do
+  @moduledoc """
+  A search backend that uses the khepri database
+  """
+  alias Lexical.Project
+  alias Lexical.RemoteControl.Search.Store.Backend
+  alias Lexical.RemoteControl.Search.Store.Backends.Khepri.Operations
+  alias Lexical.VM.Versions
+
+  require Logger
+
+  @behaviour Backend
+
+  @impl Backend
+  def new(%Project{} = project) do
+    result =
+      project
+      |> data_directory()
+      |> :khepri.start(khepri_store_id(project))
```

Comment:
Maybe worth wrapping all this in a little helper to clean up this function body. 

```elixir
case start_khepri(project) do
  {:ok, _} = success ->
    success

  ...
end
```